From: Antonio Larrosa <larrosa@kde.org>
Subject: Update requirements to python3

Remove requirements that are not required on python3
and renamed pep8 to pycodestyle
Index: mycroft-core-release-v18.2.5/requirements.txt
===================================================================
--- mycroft-core-release-v18.2.5.orig/requirements.txt
+++ mycroft-core-release-v18.2.5/requirements.txt
@@ -2,7 +2,7 @@ six==1.10.0
 websocket-client==0.32.0
 requests-futures==0.9.5
 pyalsaaudio==0.8.2
-xmlrunner==1.7.7
+#xmlrunner==1.7.7
 pyserial==3.0
 psutil==5.2.1
 pocketsphinx==0.1.0
@@ -36,4 +36,4 @@ google-api-python-client==1.6.4
 petact==0.1.2
 
 # dev setup tools
-pep8==1.7.0
+pycodestyle==1.7.0
